src/server/src/test/resources/cassandra-reaper-cassandra-at.yaml

```diff
@@ -37,6 +37,14 @@ jmxPorts:
   127.0.0.6: 7600
   127.0.0.7: 7700
   127.0.0.8: 7800
+  
+jmxCredentials:
+  "test cluster":
+    username: cassandra
+    password: cassandra
+  cluster2:
+    username: cassandra
+    password: cassandra2
```

Comment:
do we still also have a cassandra yaml (and associated test runs) on a non-jmx-secured cluster?